lib/node/Status.js

```diff
@@ -81,7 +81,7 @@ Status.childChanged = function(node, childStatus, childArgIndex=null) {
     return substeps;
   }
 
-  if (Type.isParenthesis(node)) {
+  if (node.type === 'Parentheses') {
```

Comment:
can you add parenthesis to query? don't want to check it against plain strings like that